install_config/persistent_storage/pod_security_context.adoc

```diff
@@ -158,116 +410,155 @@ Currently the list of volumes which support ownership management includes:
+    openshift.io/sa.scc.mcs: s0:c1,c0 <2>
+...
+```
+<1> _MustRunAs_ causes volume relabeling.
+<2> if the label is not provided in the pod or in the SCC then the default comes from the namespace.
+
+All predefined SCCs, except for the _privileged_ SCC, set the `seLinuxContext` to _MustRunAs_.
+This forces pods to use MCS labels, which can be defined in the pod spec, the image, or provided
+as a default.
+
+The SCC determines whether or not to require a selinux label and can provide a default label.
+If the `seLinuxContext` strategy is set to _MustRunAs_, and the pod (or image) does not
+define a label then a default, either from the SCC itself or from the namespace, is used. If
+`seLinuxContext`  is set to _RunAsAny_ then no default labels are provided, so the container
+determines the final label. In the case of docker, the container will use a unique MCS label,
+which will not likely match the labeling on existing storage mounts. Volumes which support
+SELinux management will be relabeled so that they are accessible by the specified label and,
 depending on how exclusionary the label is, only that label.
 
-This means two things:
+This means two things for unprivileged containers:
 
-* If the container is unprivileged the volume will be given a `*type*` which is
-accessible by unprivileged containers. Usually *svirt_sandbox_file_t*.
-* If a `*level*` is specified, the volume will be labeled with the given MCS
-label.
+* the volume will be given a `type` which is accessible by unprivileged containers.
+This `type` is usually *svirt_sandbox_file_t*.
+* if a `level` is specified, the volume will be labeled with the given MCS label.
 
 [NOTE]
 ====
 Level and MCS label are used interchangeably in this topic.
 ====
 
-For your volume to be accessible by your pod, the pod must have both categories
-of the volume. So a pod with *s0:c1,c2* will be able to access volumes with
-*s0,c1,c2*, and a volume with *s0* will be accessible by all pods.
-
-[WARNING]
-====
-Hard coding MCS labels into your pod definition makes it easy for others to
-determine what MCS label is needed to access the same volume as the defined pod.
-So it is especially important to rely on the MCS labels allocated by OpenShift
-and not use this option with care.
-====
-
-SELinux options are specified as follows:
-
-====
-[source,yaml]
-----
-apiVersion: v1
-kind: Pod
-metadata:
-  name: ebs-web
-spec:
-  containers:
-    - name: web
-      image: nginx
-      ports:
-        - name: web
-          containerPort: 80
-      volumeMounts:
-          - name: ebs-volume
-            mountPath: "/usr/share/nginx/html"
-  securityContext:
-    seLinuxOptions:
-    level: "s0:c123,c456"
-  volumes:
-    - name: ebs-volume
-      awsElasticBlockStore:
-      volumeID: <VOLUME ID>
-----
-====
-
-Currently the list of volumes which support SELinux management includes:
-
-* AWS Elastic Block Store
-* OpenStack Cinder
-* GCE Persistent Disk
-* iSCSI
-* emptyDir
-* Ceph RBD
-* gitRepo
```

Comment:
This is from the original doc. I did not change this list. If gitRepo is not supported I will delete it. Can we get confirmation on this @pmorie  or @pweil ?